### DISCUSSION TITLE
Is client side encryption supported for cloud storage?

### DISCUSSION THREAD

**Discussion by zubieta** [openobserve/openobserve#5086] [Q&A]
Hi! As the title says I've been trying to find out if client side encryption is possible before uploading the data to cloud storage, but I haven't been able to find anything in the docs or repo. I believe it might not be but want to confirm.

**Comment by prabhatsharma** [comment]
We are discussing this internally. Can you please create a detailed issue for this that will help guide the discussion.

**Comment by zubieta** [comment]
Sure, thank you

**Reply by zubieta** [reply]
Here is the issue #5115
